### DISCUSSION TITLE
Can I have a deb package of theia for Ubuntu

### DISCUSSION THREAD

**Discussion by llywei** [eclipse-theia/theia#8566] [General] [answered]
Can I have a deb package of theia for Ubuntu

**Comment by vince-fugnitto** [comment, marked as answer]
@llywei thank you for the discussion, and yes it is possible.

Under our [theia-apps](https://github.com/theia-ide/theia-apps) repo, we have an example [theia-electron](https://github.com/theia-ide/theia-apps/tree/master/theia-electron), which uses `electron-builder` to package the application for desktop.

You can take a look at both the example and the official documentation for `electron-builder` and the configuration possible for [Linux](https://www.electron.build/configuration/linux).
